### ISSUE TITLE
Bug: central label text disappears on some window sizes

### ISSUE THREAD

**Issue by Emasoft** [codrops/DraggableDualViewSlideshow#1]
There is a bug in this beautiful demo. The central label text disappears or turns black on some browser window sizes. I made a couple of screenshots (my browser: OS X Safari 7.0.4).

![bug1](https://cloud.githubusercontent.com/assets/713559/3401611/5233f4be-fd52-11e3-89b2-f86b5aa98a07.jpg)
![bug2](https://cloud.githubusercontent.com/assets/713559/3401610/52313f12-fd52-11e3-8d36-4d8bcbef04b5.jpg)

**Comment by crnacura** [comment]
Thanks for reporting this. I can't confirm the disappearance in Safari (Version 6.1.4, OSX 10.8.5) but I do notice a jump which probably happens due to the animation. If you replace _-webkit-animation: slideUpFade 0.4s 0.1s_ with _-webkit-animation: slideUpFade 0.4s 0.1s both;
_ do you still see that issue? It's in the _.img-dragger-small .slide h2_ rule.

**Comment by Emasoft** [comment]
Yes, I still see the issue after making the modification you suggested.
Here is a screenshot: 
![bug3jpg](https://cloud.githubusercontent.com/assets/713559/3405547/c1756606-fd7e-11e3-841b-85db6263c712.jpg)

To reproduce the bug you can toggle between fullscreen and carousel mode. When you toggle from fullscreen to carousel, you get the title disappearing in black. But when you toogle back to fullscreen, it gets immediately visible and white, even before the animated transition starts.

**Comment by crnacura** [comment]
Thanks for trying that out. I can see a very faded title, so maybe there's some conflict with the animation because of the opacity. Could you try remove the animation from the h2 completely? Thx
